hg-fast-export.py

```diff
@@ -7,6 +7,11 @@
 from hg2git import setup_repo,fixup_user,get_branch,get_changeset
```

Comment:
Corrected commit message:

When the fast-export script finds a .hglf pointer file, it will
convert it to its counterpart git lfs pointer file. To work correctly
the mercurial large file needs to be in the user's cache, otherwise 
fast export will die after providing some instructions on how to continue.